person.rb

```diff
@@ -0,0 +1,32 @@
+class Person
+  attr_reader :id, :parent_permission
+  attr_accessor :age, :name
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1...10_000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    return true if age >= 18
+
+    false
+  end
+
+  public
+
+  def can_use_services?
+    return true if of_age? || parent_permission
+
+    false
+  end
```

Comment:
[OPTIONAL] If you define the `can_use_services?` method before the `private` keyword, it will be as default a public method without adding the `public` keyword.